specs/features/landing_feature.js

```diff
@@ -0,0 +1,32 @@
+var Setup = require('../features/helpers/setup');
+
+describe('Landing page', function() {
```

Comment:
What about ES6 for tests? - I know, the ES6 syntax doesn't exclude the ES5, but we could use arrow functions and imports for tests as well, if it is not a big deal.